CMakeLists.txt

```diff
@@ -22,8 +22,43 @@ include( CTest )
   #set( CMAKE_CXX_VISIBILITY_PRESET hidden )
 
 #---------------------------------------------------------------------
+option( ELASTIX_USE_OPENCL "Use OpenCL enabled GPU" OFF )
+set(_GPU_depends "")
+if(ELASTIX_USE_OPENCL)
+  list(APPEND _GPU_depends ITKGPUCommon)
+endif()
+
 # Find ITK.
-find_package( ITK 5.1.1 REQUIRED )
+find_package( ITK 5.1.1 REQUIRED COMPONENTS
```

Comment:
We likely want to loosen the version requirement, i.e. `5.1.1` -> `5` or `5.1`.

Reply:
> We likely want to loosen the version requirement, i.e. 5.1.1 -> 5 or 5.1

@thewtex Why would you? It may change the behavior of elastix, for example when the following ITK bug fix may not be assumed https://github.com/InsightSoftwareConsortium/ITK/issues/1950

Reply:
An ITK installation may be available that is newer, e.g. 5.1.2, 5.2rc01, and we can enable use of that version.